### PR TITLE
[terra-icon] Updated eIFU text to be Instructions for Use 

### LINKED ISSUE #2759: Update eIFU text to be Instructions for Use instead of Information for Use
# Feature Request

The design standard for eIFU has been updated to "Instructions for Use" instead of "Information for Use" based on feedback from HF.  Will need to get this minor update included in any of the places where the eIFU icon is being used (About Box, etc).

https://wiki.cerner.com/x/9e2yeg

### PR DESCRIPTION
**Pull request by saket2403**
### Summary
Fixes #2759

### CODE REVIEW

File: packages/terra-icon/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Updated eIFU text to be ` Electronic Instructions for Use Icon`.
```
Comment (neilpfeiffer) [inline packages/terra-icon/CHANGELOG.md]: nit: extra space between ` ' ` and `E`
```suggestion
* Updated eIFU text to be `Electronic Instructions for Use Icon`.
```

**Comment by neilpfeiffer** [comment]
@PaulSolDev will need update to requirements

File: packages/terra-icon/CHANGELOG.md (hunk 1 above)
Comment (saket2403) [reply packages/terra-icon/CHANGELOG.md]: Changed 45049c0

**Comment by PaulSolDev** [comment]
Functional review complete. No issues identified.

**Pull request merged**